.gitignore

```diff
@@ -1,3 +1,173 @@
-.DS_Store*
-node_modules
+### Node ###
```

Comment:
"no!"? Could you elaborate on "no"? Your `.gitignore` was (is) lacking some _very common_ entries. I myself had to manually ignore Sublime files when cloning the repo. Having a somewhat standardized list of ignores files leads to easiness of contributions.

Also, `.DS_Store*` and `node_modules` are present down below.